Resources/Prototypes/Recipes/Construction/Graphs/windoor.yml

```diff
@@ -0,0 +1,228 @@
+﻿- type: constructionGraph
+  id: windoor
+  start: start
+  graph:
+  - node: start
+    edges:
+    - to: assembly
+      completed:
+      - !type:SetAnchor
+        value: false
+      steps:
+      - material: Steel
+        amount: 4
+        doAfter: 2
+    - to: assemblySecure
+      completed:
+      - !type:SetAnchor
+        value: false
+      steps:
+      - material: Plasteel
+        amount: 4
+        doAfter: 2
+
+  - node: assembly
+    entity: WindoorAssembly
+    actions:
+    - !type:SnapToGrid {}
+    - !type:SetAnchor {}
+    edges:
+    - to: glass
+      conditions:
+      - !type:EntityAnchored {}
+      steps:
+      - material: Glass
+        amount: 5
+        doAfter: 1
+    - to: start
+      conditions:
+      - !type:EntityAnchored
+        anchored: false
+      completed:
+      - !type:SpawnPrototype
+        prototype: SheetSteel1
+        amount: 4
+      - !type:DeleteEntity {}
+      steps:
+      - tool: Welding
+        doAfter: 2
+
+  - node: assemblySecure
```

Comment:
It doesn't matter too much, but I'd prefer the windoor and securewindoor construction paths to be separated instead of intermixed.